webui/react/src/routes/utils.ts

```diff
@@ -68,14 +68,18 @@ export const handlePath = (
   if (options.onClick) {
     options.onClick(event);
   } else if (href) {
-    if (event.button === 1 || event.metaKey || event.ctrlKey || options.popout) {
+    if (isShiftClick(event) || options.popout) {
       openBlank(href);
     } else {
       routeAll(href);
     }
   }
 };
 
+export const isShiftClick = (event: MouseEvent|React.MouseEvent): boolean => {
```

Comment:
**suggestion:** maybe something more prescriptive like `isNewTabEvent` or `isNewTabClick`?

Reply:
isNewTabClickEvent?

and there was another comment to move this in another util file, totally fine for me.

Reply:
yeah not sure what or which util file it should go into, but I don't feel strongly about leaving it here for now or moving it.